proc_IR.py

```diff
@@ -16,12 +16,14 @@
 # }}}
 
 for searchstr,exp_type,nodename, postproc in [
-        ('200303_IR_AER_6','test_equip','signal','spincore_IR'),
+        ('200212_IR_3_30dBm', 'test_equip', 'signal', 
+            'spincore_IR'),
         ]:
     s = find_file(searchstr, exp_type=exp_type,
             expno=nodename,
             postproc=postproc, lookup=postproc_dict,
             dimname='indirect')
+    s *= exp(-1j*s.fromaxis('indirect')*clock_correction)
```

Comment:
clock correction should be part of the list of tuples above

Reply:
we would have to adjust the find_file function as clock_correction is not an argument of it currently.

Reply:
would you rather keep it as is and have clock_correction as a variable we change upon processing or incorporate it into pyspecdata?

Reply:
that doesn't make sense -- I suggest a fix below in 2cbc803971b64e87881

Reply:
my bad I misunderstood! Got it now though!